evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -53,6 +53,7 @@ def next_batch(self):
         Returns:
             list(PipelineBase): a list of instances of PipelineBase subclasses, ready to be trained and evaluated.
         """
+        run_ensembling = self.ensembling and len(self.allowed_pipelines) > 1
```

Comment:
nit-pick: Why not make `self.ensembling = ensembling and len(self.allowed_pipelines) > 1` in the init instead of defining `run_ensembling`?